README.md

```diff
@@ -69,8 +69,8 @@ Mastodon acts as an OAuth2 provider, so 3rd party apps can use the REST and Stre
 
 - **PostgreSQL** 12+
 - **Redis** 4+
-- **Ruby** 3.1+
-- **Node.js** 16+
+- **Ruby** 3.2+
```

Comment:
In line with [`.ruby-version`](https://github.com/mastodon/mastodon/blob/85fdbd0ad53837c9209acf3fb45811d5bae41cd9/.ruby-version#L1).

Reply:
Reverted: https://github.com/mastodon/mastodon/compare/9726a61e255dd7853ed46e08c3a20d95887d338f..702e29d3879b26d53cd503d5c22684d80af5a42c